rest_framework/response.py

```diff
@@ -81,7 +80,7 @@ def status_text(self):
         """
         # TODO: Deprecate and use a template tag instead
         # TODO: Status code text for RFC 6585 status codes
-        return STATUS_CODE_TEXT.get(self.status_code, '')
+        return httplib[self.status_code]
```

Comment:
Where is it suggested by Django?

Also, what benefit does this bring that the old `STATUS_CODE_TEXT` dictionary didn't provide?